packages/tree-select/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
we need to keep the package-lock for each package (unlike gutenberg) to make `npm ci` happy